mac/Keyman4MacIM/Keyman4MacIM/KMAboutWindow/KMAboutWindowController.m

```diff
@@ -51,7 +51,11 @@ - (IBAction)configAction:(id)sender {
     // High Sierra (10.13.1 - 10.13.3). See: https://bugreport.apple.com/web/?problemID=35422518
     // TODO: This bug should be fixed in 10.13.4, so after that has been available for a few weeks, we
     // can create a PR for branch mac-revert-high-sierra-showPreferences-workaround and merge it.
-    [self.AppDelegate showConfigurationWindow];
+    // rrb: where available (everywhere but 10.13.1-10.13.3?) call showPreferences, otherwise, call our workaround
+    if ([self.AppDelegate.inputController respondsToSelector:@selector(showPreferences:)])
```

Comment:
Have you tested this on 10.13.1? I have a sneaking feeling that it responds to the selector but doesn't actually do anything. (I could be wrong -- it's been a while since we've looked at this)

Reply:
I've tested on 10.13.3, reproduced the bug (it crashes, and Fabric reports it). I've pushed, but now I'm re-checking my fixed code to make sure it does NOT crash. (I thought I did already, but I was late and it was tired last night.)

Reply:
Yes, I can confirm that my fix avoids Apple's API and calls our workaround on 10.13.3.